src/EditorFeatures/CSharpTest/Classification/SemanticClassifierTests.cs

```diff
@@ -3571,14 +3571,10 @@ void Goo()
 Namespace("Text"),
 Namespace("RegularExpressions"),
 Keyword("var"),
-Regex.Text(@"
-            "),
 Regex.Anchor("$"),
 Regex.OtherEscape("\\"),
 Regex.OtherEscape("a"),
-Regex.Comment("(?#comment)"),
-Regex.Text(@"
-            "));
+Regex.Comment("(?#comment)"));
```

Comment:
these changes are desirable.  Those whitespace chunks are not part of hte actual content of hte string (the lanugage/compiler does not include it), so it was wrong that we were producing regex classifications for it (even if it was just whitespace).  We now appropriatley only classify the actual content.